docs/meshstack.building-blocks.aws-reference-architecture.md

```diff
@@ -0,0 +1,82 @@
+---
+id: meshstack.building-blocks.permission-delegation-aws
+title: Permission Delegation on AWS
+---
+
+Building blocks are often managing resources in tenants that are owned by application teams.
+As with any high privileged access, this access needs to be handled with great care.
+
```

Comment:
Building blocks are often managing resources in tenants that are owned by application teams. Such resource management might require privileged access and needs to be handled with great care.